package/cfg/rke2-cis-1.5/master.yaml

```diff
@@ -0,0 +1,1081 @@
+          - flag: "root:root"
+            compare:
+              op: eq
+              value: "root:root"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chown root:root /etc/kubernetes/scheduler.conf
+      scored: true
+      type: skip
+
+    - id: 1.1.17
+      text: "Ensure that the controller-manager.conf file permissions are set to 644 or more restrictive (Scored)"
+      audit: "/bin/sh -c 'if test -e /etc/kubernetes/controller-manager.conf; then stat -c %a /etc/kubernetes/controller-manager.conf; fi'"
+      tests:
+        test_items:
+          - flag: "644"
+            compare:
+              op: eq
+              value: "644"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chmod 644 /etc/kubernetes/controller-manager.conf
+      scored: true
+      type: skip
+
+    - id: 1.1.18
+      text: "Ensure that the controller-manager.conf file ownership is set to root:root (Scored)"
+      audit: "/bin/sh -c 'if test -e /etc/kubernetes/controller-manager.conf; then stat -c %U:%G /etc/kubernetes/controller-manager.conf; fi'"
+      tests:
+        test_items:
+          - flag: "root:root"
+            compare:
+              op: eq
+              value: "root:root"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chown root:root /etc/kubernetes/controller-manager.conf
+      scored: true
+      type: skip
+
+    - id: 1.1.19
+      text: "Ensure that the Kubernetes PKI directory and file ownership is set to root:root (Scored)"
+      audit: "ls -laR /etc/kubernetes/pki/"
+      type: "manual"
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chown -R root:root /etc/kubernetes/pki/
+      scored: true
+      type: skip
+
+    - id: 1.1.20
+      text: "Ensure that the Kubernetes PKI certificate file permissions are set to 644 or more restrictive (Scored) "
+      audit: "stat -c %n\ %a /etc/kubernetes/pki/*.crt"
+      type: "manual"
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chmod -R 644 /etc/kubernetes/pki/*.crt
+      scored: true
+      type: skip
+
+    - id: 1.1.21
+      text: "Ensure that the Kubernetes PKI key file permissions are set to 600 (Scored)"
+      audit: "stat -c %n\ %a /etc/kubernetes/pki/*.key"
+      type: "manual"
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chmod -R 600 /etc/kubernetes/pki/*.key
+      scored: true
+      type: skip
```

Comment:
See comments for 1.1.19. shouldnt be skipped and audit and remediation need updated